pipeline_games/transform_games.py

```diff
@@ -58,25 +58,32 @@ def explode_column_to_individual_rows(data: pd.DataFrame, column_name: str) -> p
 
     data_frame = pd.read_csv('games.csv')
 
-    unique_tags_df = identify_unique_tags(data_frame)
-    user_generated_df = create_user_generated_column(unique_tags_df)
+    unique_genre_df = identify_unique_genre(data_frame)
+    user_generated_df = create_user_generated_column(unique_genre_df)
     data_frame_no_genres = drop_unnecessary_columns(
         user_generated_df, 'genres')
-    data_with_unique_tags_only = drop_unnecessary_columns(
+    data_with_unique_genre_only = drop_unnecessary_columns(
         data_frame_no_genres, 'user_tags')
 
-    data_with_unique_tags_only['release_date'] = data_with_unique_tags_only['release_date'].apply(
+    data_with_unique_genre_only['release_date'] = data_with_unique_genre_only['release_date'].apply(
         convert_date_to_datetime)
 
-    data_with_unique_tags_only['full price'] = data_with_unique_tags_only['full price'].apply(
+    data_with_unique_genre_only['full_price'] = data_with_unique_genre_only['full_price'].apply(
         convert_price_to_float)
 
-    data_with_unique_tags_only['sale price'] = data_with_unique_tags_only['sale price'].apply(
+    data_with_unique_genre_only['sale_price'] = data_with_unique_genre_only['sale_price'].apply(
         convert_price_to_float)
 
     unique_developers = explode_column_to_individual_rows(
-        data_with_unique_tags_only, 'developers')
+        data_with_unique_genre_only, 'developers')
     final_df = explode_column_to_individual_rows(
         unique_developers, 'publishers')
 
-    final_df.to_csv('transformed_data.csv')
+    final_df.to_csv('genres.csv')
+
+    games_only_df = drop_unnecessary_columns(final_df, 'genre')
+    games_df = drop_unnecessary_columns(games_only_df, 'user_generated')
+    games_df = drop_unnecessary_columns(games_df, 'developers')
+    games_df = drop_unnecessary_columns(games_df, 'publishers')
```

Comment:
drop unnecessary columns looks like can be looped with the 2nd positional argument values looped:

for column in ["genre",.....]:
    games_df - drop_unnecessary_columns(games_df, column)